server_src/routes/plugins/octoprintRegistration.js

```diff
@@ -0,0 +1,165 @@
+const express = require("express");
+const nodeFetch = require("node-fetch");
+const OctoprintRegistrationService = require("../../services/plugins/octoprint-registration.service");
+const { Runner } = require("../../runners/state");
+const { ensureAuthenticated } = require("../../config/auth");
+const { Validator } = require("node-input-validator");
+const router = express.Router();
+
+async function handleInputValidation(req, rules, res) {
+  const v = new Validator(req.body, rules);
+  const matched = await v.check();
+  if (!matched) {
+    res.status(422).send(v.errors);
```

Comment:
@NotExpectedYet this is what I meant with returning a 4xx code on input validation. I must admit I'd rather use 400, so this is just copied over from the example I found.